TACC/protx-dashboard
# Change sizing of chart labels of high, medium, low risk for analytics…

## Overview: ##
Change sizing of chart labels of high, medium, low risk for analytics main chart. Adjusted sizes of captions and instructions.
## Related Jira tickets: ##

* 

## Summary of Changes: ##
Made risk levels on analytics histogram smaller. Adjusted classes to make the two analytics charts  cohesive and help elements line up with each other when a county is selected. 

## Testing Steps: ##
1. Go to [https://cep.test/protx/dash/analytics](https://cep.test/protx/dash/analytics)
2. Make sure histogram risk overlays are smaller and look correct.
3. Make sure tables, titles, and annotations line up. 

## UI Photos:
![Screen Shot 2022-10-25 at 2 11 52 PM](https://user-images.githubusercontent.com/96220951/197861399-003c3759-fb1a-447b-aed2-1dff07b9aadc.png)

![Screen Shot 2022-10-25 at 2 12 30 PM](https://user-images.githubusercontent.com/96220951/197861466-142581ef-3eca-4c39-b340-2adeb5ed0435.png)



## Notes: ##
